### PR TITLE
DO-439-Tab Styling

### PR DESCRIPTION
**Pull request by Palakagrawal28**

### CODE REVIEW

File: src/components/member/BusinessDetails/BusinessDetails.jsx [hunk 1]
```diff
@@ -25,11 +25,11 @@ import IconWebsite from '../../common/Typography/Icons/IconWebsite';
  * @return {Component}
  * @constructor
  */
-const BusinessDetailsSectionHeading = () => (
-  <Container className="business-details-section-heading__container">
-    <h2>Business Details</h2>
-  </Container>
-);
+// const BusinessDetailsSectionHeading = () => (
```
Comment (mohitsaladiya) [inline src/components/member/BusinessDetails/BusinessDetails.jsx:28]: Remove the commented out code

File: src/components/member/BusinessDetails/BusinessDetails.jsx [hunk 2]
```diff
@@ -573,7 +573,7 @@ const BusinessDetails = (props) => {
         className="business-details__section"
       >
         <BusinessDetailsEditLink isEdit={isEdit} />
-        <BusinessDetailsSectionHeading />
+        {/*<BusinessDetailsSectionHeading />*/}
```
Comment (mohitsaladiya) [inline src/components/member/BusinessDetails/BusinessDetails.jsx:576]: Here too.

File: src/components/member/BusinessDetails/BusinessDetails.jsx (hunk 1 above)
Comment (Palakagrawal28) [reply src/components/member/BusinessDetails/BusinessDetails.jsx:28]: Done

File: src/components/member/BusinessDetails/BusinessDetails.jsx (hunk 2 above)
Comment (Palakagrawal28) [reply src/components/member/BusinessDetails/BusinessDetails.jsx:576]: Done

**Pull request merged**